core/src/test/java/org/pentaho/reporting/platform/plugin/JobManagerContextTest.java

```diff
@@ -53,15 +52,23 @@ public class JobManagerContextTest {
   private UUID uuid;
   private IPentahoSession session;
 
+  private MockedStatic<ReportCreator> mockStaticReportCreator;
+
+  private MockedStatic<PentahoSessionHolder> mockStaticPentahoSessionHolder;
+
   @Before
   public void before() throws ResourceException, IOException {
+    mockStaticReportCreator = mockStatic( ReportCreator.class );
+    mockStaticPentahoSessionHolder = mockStatic( PentahoSessionHolder.class );
     session = mock( IPentahoSession.class );
     uuid = UUID.randomUUID();
     final IPentahoAsyncExecutor executor = mock( IPentahoAsyncExecutor.class );
     state = mock( IAsyncReportState.class );
     report = mock( MasterReport.class );
     when( executor.getReportState( uuid, session ) ).thenReturn( state );
     PentahoSystem.registerObject( executor, IPentahoAsyncExecutor.class );
+    mockStaticReportCreator.when(  () -> ReportCreator.createReportByName( any() ) ).thenReturn( report );
+    mockStaticPentahoSessionHolder.when( PentahoSessionHolder::getSession ).thenReturn( session );
```

Comment:
Why did all of these get moved to the setup method?  They're far better off where they were in the try-with-resources block to ensure they don't interfere with any other tests.